report-03/threat model.wiki

```diff
@@ -117,6 +141,10 @@ This document outlines the OBPP threat model used to develop [[criteria.md|crite
 |Use mixing protocols which are secure against misbehavior by any participant
 |I D 3 a <br/> III B 1 a
 |-
+|Correlate inputs and outputs to an identified individual as part of a multisig signing process
```

Comment:
This appears to be an attack for issue #53. At present, it is worded somewhat vaguely.